arm-sql/2014-04-01/swagger/dataMasking.json

```diff
@@ -0,0 +1,460 @@
+    "info": {
+        "title": "Azure Sql Database API spec",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/dataMaskingPolicies/Default": {
+            "put": {
+                "tags": [
+                    "DataMasking"
+                ],
+                "operationId": "DataMasking_CreateOrUpdatePolicy",
+                "description": "Creates or updates an Azure SQL Database data masking policy",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameter"
+                    },
+                    {
+                        "name": "databaseName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL Database for which the data masking rule applies."
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/DataMaskingPolicy"
+                        },
+                        "description": "The required parameters for createing or updating a firewall rule."
+                    }
+                ],
+                "responses": {
+                    "201": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/DataMaskingPolicy"
+                        }
+                    },
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/DataMaskingPolicy"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "DataMasking"
+                ],
+                "operationId": "DataMasking_GetPolicy",
```

Comment:
I would prefer to keep all DataMasking APIs in the DataMasking operation group rather than giving each its own operation group.

---
In reply to: [80145813](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80145813) [](ancestors = 80145813)